homeassistant/components/tado/sensor.py

```diff
@@ -183,6 +195,16 @@ def get_geofencing_mode(data) -> str:
     state_fn=lambda data: data.ac_power,
     attributes_fn=lambda data: {"time": data.ac_power_timestamp},
 )
+COMFORT_TEMPERATURE_ENTITY_DESCRIPTION = TadoSensorEntityDescription(
+    key="air comfort temperature",
+    translation_key="air_comfort_temperature",
+    state_fn=lambda data: data["temperatureLevel"],
+)
+COMFORT_HUMIDITY_ENTITY_DESCRIPTION = TadoSensorEntityDescription(
```

Comment:
I'm guessing this is a sensor for humidity. I think we should add the native unit of measurement and the device class humidity

Reply:
see above